go.mod

```diff
@@ -56,7 +56,7 @@ require (
 	github.com/golang/protobuf v1.5.3 // indirect
 	github.com/golang/snappy v0.0.4 // indirect
 	github.com/google/pprof v0.0.0-20211214055906-6f57359322fd // indirect
-	github.com/klauspost/compress v1.15.11 // indirect
+	github.com/klauspost/compress v1.17.3 // indirect
```

Comment:
These changed in here due to https://github.com/couchbase/go-blip/pull/68 - this is the first time bumping `go-blip` since